packages/circuit-ui/components/Checkbox/__snapshots__/Checkbox.spec.tsx.snap

```diff
@@ -238,23 +265,34 @@ exports[`Checkbox Styles should render with default styles 1`] = `
   border-color: var(--cui-border-normal);
 }
 
-.circuit-1:checked:focus:not(:focus-visible)+label::before {
+.circuit-1:checked:focus:not(:focus-visible)+label::before,
+.circuit-1:indeterminate:focus:not(:focus-visible)+label::before {
   border-color: var(--cui-border-accent);
 }
 
-.circuit-1:checked+label>svg {
+.circuit-1:checked+label>svg[data-symbol='checked'],
+.circuit-1:indeterminate+label>svg[data-symbol='indeterminate'] {
   -webkit-transform: translateY(-50%) scale(1, 1);
   -moz-transform: translateY(-50%) scale(1, 1);
   -ms-transform: translateY(-50%) scale(1, 1);
   transform: translateY(-50%) scale(1, 1);
   opacity: 1;
 }
 
-.circuit-1:checked+label::before {
+.circuit-1:checked+label::before,
+.circuit-1:indeterminate+label::before {
   border-color: var(--cui-border-accent);
   background-color: var(--cui-bg-accent-strong);
 }
 
+.circuit-1:checked:disabled+label::before,
+.circuit-1:checked[disabled]+label::before,
+.circuit-1:indeterminate:disabled+label::before,
+.circuit-1:indeterminate[disabled]+label::before {
+  border-color: var(--cui-border-accent-disabled);
+  background-color: var(--cui-bg-accent-strong-disabled);
+}
```

Comment:
otherwise, disabled colors are applied, meaning white on white, and you can't see the checkboxes.

I added the stories to see on storybook & chromatic the 'disabled+checked', 'disabled+invalid+checked', 'invalid+indeterminate', etc...